pages/index.js

```diff
@@ -9,94 +9,55 @@ import { useDispatch, useSelector } from "react-redux";
 import { getAllMusic } from "../actions/musicActions";
 
 const HomePage = () => {
-  const dispatch = useDispatch();
-  useEffect(() => {
-    dispatch(getAllMusic());
-  }, []);
-  return (
-    <div>
-      <Head>
-        <link
-          rel="apple-touch-icon"
-          sizes="57x57"
-          href="/apple-icon-57x57.png"
-        />
-        <link
-          rel="apple-touch-icon"
-          sizes="60x60"
-          href="/apple-icon-60x60.png"
-        />
-        <link
-          rel="apple-touch-icon"
-          sizes="72x72"
-          href="/apple-icon-72x72.png"
-        />
-        <link
-          rel="apple-touch-icon"
-          sizes="76x76"
-          href="/apple-icon-76x76.png"
-        />
-        <link
-          rel="apple-touch-icon"
-          sizes="114x114"
-          href="/apple-icon-114x114.png"
-        />
-        <link
-          rel="apple-touch-icon"
-          sizes="120x120"
-          href="/apple-icon-120x120.png"
-        />
-        <link
-          rel="apple-touch-icon"
-          sizes="144x144"
-          href="/apple-icon-144x144.png"
-        />
-        <link
-          rel="apple-touch-icon"
-          sizes="152x152"
-          href="/apple-icon-152x152.png"
-        />
-        <link
-          rel="apple-touch-icon"
-          sizes="180x180"
-          href="/apple-icon-180x180.png"
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="192x192"
-          href="/android-icon-192x192.png"
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="32x32"
-          href="/favicon-32x32.png"
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="96x96"
-          href="/favicon-96x96.png"
-        />
+    const links = [
+        { sizes: "57x57", href: "/apple-icon-57x57.png" },
+        { sizes: "60x60", href: "/apple-icon-60x60.png" },
+        { sizes: "72x72", href: "/apple-icon-72x72.png" },
+        { sizes: "76x76", href: "/apple-icon-76x76.png" },
+        { sizes: "114x114", href: "/apple-icon-114x114.png" },
+        { sizes: "120x120", href: "/apple-icon-120x120.png" },
+        { sizes: "144x144", href: "/apple-icon-144x144.png" },
+        { sizes: "152x152", href: "/apple-icon-152x152.png" },
+        { sizes: "180x180", href: "/apple-icon-180x180.png" },
+        { sizes: "192x192", href: "/android-icon-192x192.png" },
+        { sizes: "32x32", href: "/favicon-32x32.png" },
+        { sizes: "96x96", href: "/favicon-96x96.png" },
+        { sizes: "16x16", href: "/favicon-16x16.png" },
+    ];
+
```

Comment:
instead of keeping this here we can create a directory under src with name `resources` having index.js file only  and from there we can export these values to index.js